velox/common/memory/AllocationPool.h

```diff
@@ -35,7 +35,7 @@ class AllocationPool {
 
   void clear();
 
-  char* allocateFixed(uint64_t bytes);
+  char* allocateFixed(uint64_t bytes, int alignment = 1);
```

Comment:
int -> int32_t

Would you document this method and add a test? Consider extracting this change into a separate PR.